README.md

```diff
@@ -35,6 +35,22 @@ To use N-API in a native module:
 #include "napi.h"
 ```
 
+To make the migration to node-api easier, we have also provide a script to do these steps above for you. To use the conversion script:
```

Comment:
"we have also provide" -> "we have provided"
"to do these steps above for you" -> "to help complete the steps listed above"